pages/common/rustup.md

```diff
@@ -1,8 +1,8 @@
 # rustup
 
-> Rust toolchain installer.
 > Install, manage, and update Rust toolchains.
-> More information: <https://github.com/rust-lang/rustup.rs>.
+> Some subcommands, such as `toolchain` `target`, `update`, etc. have their own usage documentation, accessible via `tldr rustup subcommand`.
```

Comment:
```suggestion
> Some subcommands, such as `toolchain` `target`, `update`, etc. have their own usage documentation, accessible via `tldr rustup <subcommand>`.
```

Reply:
That's how it's written in the `git` and `cs` pages.

https://github.com/tldr-pages/tldr/blob/main/pages/common/git.md?plain=1#L4
```markdown
> Some subcommands such as `commit`, `add`, `branch`, `checkout`, `push`, etc. have their own usage documentation, accessible via `tldr git subcommand`.
```

https://github.com/tldr-pages/tldr/blob/main/pages/common/cs.md?plain=1#L4
```markdown
> Some subcommands such as `install`, `launch`, `java`, `fetch`, `resolve`, `complete-dep`, etc. have their own usage documentation, accessible via `tldr cs subcommand`.
```

Should that be changed too?
Or maybe removed altogether, there's many more pages with this kind of notice but without the second part:

```suggestion
> Some subcommands, such as `toolchain`, `target`, `update`, etc. have their own usage documentation.
```

Reply:
Yeah, we can remove it altogether, the latter LGTM.